api/server/server.go

```diff
@@ -1058,14 +1058,11 @@ func (s *Server) postContainersWait(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	name := vars["name"]
-	cont, err := s.daemon.Get(name)
-	if err != nil {
+	status, err := s.daemon.ContainerWait(vars["name"], -1*time.Second)
+	if err != nil && !strings.Contains(strings.ToLower(err.Error()), "timed out") {
```

Comment:
Unless someone can come up with a good reason I wouldn't ignore the error, let's just return it.  I think the only error we could get in this case is the "container not found" one, which I think is valid to return.